### PR TITLE
Docs: add code style requirements

### PR DESCRIPTION
**Pull request by krisszzzz**
Add code style requirements that should be used in the project

### CODE REVIEW

File: code_style.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        first_parameter, second_parameter, third_parameter_p );
+return;
+}
+```
+
+### Control statements
+Control statement parentheses should be spaced, brace should be newlined:
+```cpp
+// Note: else if / else should be on same level with '}'
+if ( condition_1 )
```
Comment (NikitaDzer) [inline code_style.md:82]: Change that example with
```c
if ( condition_1 )
{
    // Your code.
} else if ( condition_2 )
{
    // Your code.
} else
{
    // Your code.
}
```

Comment (NikitaDzer) [reply code_style.md:82]: Leave the comment `// Your code.` where it's necessary.

File: code_style.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+/*
+Brief function description
+*/
+
+static void
+funcName( int first_parameter, /* comment for first parameter */
+          long second_parameter, /* comment for second parameter */
+          char* p_third_parameter ) /* comment for third parameter */
+{
+printf( "%d %ld %s\\n",
```
Comment (NikitaDzer) [inline code_style.md:72]: Add tabs there.

File: code_style.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        first_parameter, second_parameter, third_parameter_p );
+return;
```
Comment (NikitaDzer) [inline code_style.md:74]: Do we need to put cringe `return;` here?)

**Review by NikitaDzer — COMMENTED** [review]
Please, check comments.

File: code_style.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+Every header file should be protected from double include:
+
+```cpp
+
+#ifndef FILE_NAME_H
+#define FILE_NAME_H
+/**
+ * file content.
+ */
+#endif /* FILE_NAME_H */
```
Comment (serjzimmerman) [inline code_style.md:35]: Replace the include guard with

```
#pragma once

/*
 * File content.
 */ 
```

**Review by serjzimmerman — APPROVED** [review]

**Review by NikitaDzer — APPROVED** [review]
The first pull request, congratulations!

**Pull request merged**